### ISSUE TITLE
Chrome reports sharedwithexpose.com as deceptive site

### ISSUE THREAD

**Issue by mwkcoding** [beyondcode/expose#277]
This seems like an error. But very annoying.

**Comment by nspaul** [comment]
I'm seeing similar behavior. Yesterday I was able to share my site via my Expose subdomain, and at the end of the day I just used Ctrl-C to close the tunnel connection. I started and restarted this many times yesterday with no issues. This morning, I hit the up arrow in my terminal to restart the tunnel, and I get this error today:
`Connection to sharedwithexpose.com:443 failed during TLS handshake: Unable to complete TLS handshake: SSL operation failed with code 1. OpenSSL Error messages: error:140943E8:SSL routines:ssl3_read_bytes:reason(1000)`
I restarted my machine, and at first I was able to share the site just fine, but my browsers gave me the "deceptive site" error. Then after I tried sharing again I get the same error in my terminal.

I'm really not sure if this is some issue on my end, or if something is going on with the Expose infrastructure. I just wanted to share my experience here in case it helps to tell the greater story (and if someone has a resolution, I'm all ears!)
